databuilder/databuilder/extractor/generic_usage_extractor.py

```diff
@@ -0,0 +1,111 @@
+"""
+The Generic Usage Extractor allows you to populate the "Frequent Users" and "Popular Tables" features within
+Amundsen with the help of the TableColumnUsage class. Because this is a generic usage extractor, you need to create
+a custom log parser to calculate poplarity. There is an example of how to calculate table popularity for Snowflake
+in databuilder/example/scripts/sample_snowflake_table_usage.scala.
+"""
+
+import logging
+from typing import (
+    Any, Dict, Iterator, Union,
+)
+
+from pyhocon import ConfigFactory, ConfigTree
+
+from databuilder import Scoped
+from databuilder.extractor import sql_alchemy_extractor
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.sql_alchemy_extractor import SQLAlchemyExtractor
+from databuilder.models.table_column_usage import ColumnReader, TableColumnUsage
+
+LOGGER = logging.getLogger(__name__)
+
+
+class GenericUsageExtractor(Extractor):
+    # SELECT statement from table that contains usage data
+    SQL_STATEMENT = """
+        SELECT
+            database,
+            schema,
+            name,
+            user_email,
+            read_count
+        FROM
+            {database}.{schema}.{table}
+        {where_clause_suffix};
+    """
+    # CONFIG KEYS
+    DATABASE_DRIVER = ''
```

Comment:
should that remain empty? if that's the property for sql alchemy connection the name could be misleading too

Reply:
Oops sorry! Replaced this with `DATABASE_KEY`